.env.development.local

```
@@ -1,2 +1,2 @@
 REACT_APP_MOCK_DATA=0
-REACT_APP_BACKEND_URL=http://127.0.0.1:8000/api/
+REACT_APP_BACKEND_URL=http://127.0.0.1:8000/
```

We agreed to use port 8080 in our containers

```suggestion
REACT_APP_BACKEND_URL=http://127.0.0.1:8080/
```